src/main/java/com/example/authservice/messaging/MQConfig.java

```diff
@@ -0,0 +1,49 @@
+package com.example.authservice.messaging;
+
+import org.springframework.amqp.core.*;
+import org.springframework.amqp.rabbit.connection.ConnectionFactory;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.amqp.support.converter.Jackson2JsonMessageConverter;
+import org.springframework.amqp.support.converter.MessageConverter;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class MQConfig {
+
+    public static final String QUEUE = "message_queue";
+    public static final String EXCHANGE = "message_exchange";
+    public static final String ROUTING_KEY = "message_exchange";
+
+    @Bean
+    public Queue queue(){
```

Comment:
Are we using the queue?

Reply:
If we don't need a message queue for this service then we can remove it. 
It was added in the example I followed and I'm not certain how our service is going to be used when we put all services together, so I think I need to figure that out first before we remove it.